lib/core/versions/latest/Multihash.ts

```diff
@@ -65,8 +65,9 @@ export default class Multihash {
     const contentBuffer = JsonCanonicalizer.canonicalizeAsBuffer(content);
 
     // Double hash.
-    const intermediateHashBuffer = Multihash.hashAsNonMultihashBuffer(contentBuffer, ProtocolParameters.hashAlgorithmInMultihashCode);
-    const multihashEncodedString = Multihash.hashThenEncode(intermediateHashBuffer, ProtocolParameters.hashAlgorithmInMultihashCode);
+    const hashAlgorithmInMultihashCode = 18; // Default to SHA256.
```

Comment:
Should we make this an optional param to make future test changes potentially easier?

Reply:
I considered it but decided against it, since it there is no need for it right now and it is easy to change when we need it.